config.js

```diff
@@ -3,188 +3,35 @@ const templateData =
-                link: ""
-              },
-              {
-                title: "timer",
-                link: ""
-              },
-              {
-                title: "redirect",
-                link: ""
-              },
-              {
-                title: "copy",
-                link: ""
-              },
-              {
-                title: "pause",
-                link: ""
-              }
-            ]
-          },
-          {
-            title: "Functions",
-            link: "",
-            sub_items: [
-              {
-                title: "clear cookies",
-                link: ""
-              }
-            ]
-          },
-          {
-            title: "Recording",
-            link: ""
-          },
-          {
-            title: "Play",
-            link: ""
-          },
-          {
-            title: "Clipboard",
-            link: ""
-          },
-          {
-            title: "Options",
-            link: "",
-            sub_items: [
-              {
-                title: "CBA",
-                link: ""
-              },
-              {
-                title: "Functions Option",
-                link: ""
-              },
-              {
-                title: "Import-Export",
-                link: ""
-              }
-            ]
-          }
-        ]
-      },
-      {
-        title: "Tutorial",
-        link: "#",
-        sub_items: [
-          {
-            title: "Registration Demo",
-            link: ""
-          },
-          {
-            title: "Facebook Demo",
-            link: ""
-          },
-          {
-            title: "Recording",
-            link: ""
-          },
-          {
-            title: "Video Presentation for CBA",
-            link: ""
-          }
-        ]
-      },
-      {
-        title: "Ready Projects",
-        link: "#",
-        sub_items: [
-          {
-            title: "FB like/unlike",
-            link: ""
-          }
-        ]
-      },
+    menuItems: ["", "documentation", "tutorial", "ready-projects",
+      "what's-new"],
+    listOfPages: function (pathname)
+    {
+      const query = templateData.site.queryPages;
+      const {originalPathname} = query((page) => page.pathname === pathname)[0];
+
+      let pagesQuery = query((page) => page.originalPathname.startsWith(originalPathname) &&
+        page.originalPathname.replace(originalPathname, "").split("/").length === 2);
+
+      pagesQuery.sort((a, b) =>
       {
-        title: "What's New",
-        link: "#",
-        sub_items: [
-          {
-            title: "Version 8.0",
-            link: ""
-          },
-          {
-            title: "Version 6.0",
-            link: ""
-          },
-          {
-            title: "Version 5.0",
-            link: ""
-          }
-        ]
-      }
-    ]
+        if (!a.weight)
+          return 1;
+        if (!b.weight)
+          return -1;
+        return a.weight - b.weight;
+      });
```

Comment:
Suggestion: We are duplicating current function, I think we can reuse it.

We can introduce new method in templateData, maybe ex.: `sortByWeight` and use it both here and in `navigation.ejs`.